keps/sig-network/20190603-EndpointSlice-API.md

```diff
@@ -372,6 +430,32 @@ Based on the data collected from user clusters, vast majority (> 99%) of the k8s
 
 The current Endpoints API only includes a boolean state (Ready vs. NotReady) on individual endpoint. However, according to pod life cycle, there are more states (e.g. Graceful Termination, ContainerReary). In order to represent additional states other than Ready/NotReady,  a status structure is included for each endpoint. More condition types can be added in the future without compatibility disruptions. As more conditions are added, different consumer (e.g. different kube-proxy implementations) will have the option to evaluate the additional conditions. 
 
+- #### Why not use a CRD?
+
+**1. Protobuf is more efficient** Currently CRDs don't support protobuf. In our
```

Comment:
@wojtek-t FYI

This is unfortunate.  I was hoping we could force this to embrace CRD.